var/spack/repos/builtin/packages/totalview/package.py

```diff
@@ -3,184 +3,61 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
-import os
-
 from spack.package import *
 
 
 class Totalview(Package):
     """Totalview parallel debugger.
 
-    Source must be made available to Spack
-    externally, either by having the tarballs in the current working directory
-    or having the tarballs in a Spack mirror.
-
-    The documentation tarball will
-    be used as the primary source and the architecture appropriate tarball will
-    be downloaded as a resource."""
+    Select the version associated with your machine architecture'
+    '."""
 
     homepage = "https://totalview.io"
-    maintainers("dshrader", "petertea")
-    manual_download = True
+    maintainers("dshrader", "petertea", "suzannepaterno", "elliebinkley")
     license_required = True
     license_comment = "#"
-    license_files = ["licenses/license.dat", "licenses/tv_license_file"]
-    license_vars = ["LM_LICENSE_FILE", "TV_LICENSE_FILE"]
+    license_files = ["tv_license/license.lic"]
+    license_vars = ["RLM_LICENSE"]
```

Comment:
Are we worried about changing the expected/allowed license files? Users who have a working installation will not be able to 'simply' update their existing license file. They will have to look into the details of this package recipe to figure out what is needed. I think *extending* the set of files and vars is preferred here.